docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -94,7 +94,18 @@ The Keycloak pod replicas.
 Usualy two for the redundancy during the maintenance.
 
 
-== `tls.provider`
+== `tls`
+
+[horizontal]
+type:: dict
+default:: _see below_
```

Comment:
If this section is just there as an "introduction" to other `tls.*` parameters, I guess we can remove this?
```suggestion
```